e2e/wdio/test/helpers/edit-configs.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+
+class EditConfigs {
+  editWebConfig(filepath, serverSideRenderingEndpointUrl) {
+    let webConfig = fs.readFileSync(filepath, 'utf8');
+    let serverSideRenderingEndpoint = webConfig.match(
+      /(?<=serverSideRenderingEngineEndpointUrl=")(.*)(?=")/g
+    )[0];
+    webConfig = webConfig.replace(serverSideRenderingEndpoint, serverSideRenderingEndpointUrl);
+    fs.writeFileSync(filepath, webConfig);
+  }
+
+  editDotEnv({ filepath, jssEditingSecret, apiKey, graphQLEndpoint, rootItemPath } = {}) {
```

Comment:
Same here, could make this more generic (e.g. passing in variable name and value)